src/main/java/frc/utils/digitalinput/channeled/ChanneledDigitalInput.java

```diff
@@ -0,0 +1,32 @@
+package frc.utils.digitalinput.channeled;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+public class ChanneledDigitalInput implements IDigitalInput {
+
+	private final DigitalInput digitalInput;
+	private final Debouncer debouncer;
+	private final boolean inverted;
+
+	public ChanneledDigitalInput(int channel, double debounceTime, boolean inverted) {
+		this.digitalInput = new DigitalInput(channel);
+		this.debouncer = new Debouncer(debounceTime);
+		this.inverted = inverted;
+	}
+
+	public ChanneledDigitalInput(int channel, double debounceTime) {
+		this.digitalInput = new DigitalInput(channel);
+		this.debouncer = new Debouncer(debounceTime);
+		this.inverted = false;
+	}
```

Comment:
call the other contructor
```suggestion
	public ChanneledDigitalInput(int channel, double debounceTime) {
	    this(channel, debounceTime, false);
	}
```